internal/mesh/internal/controllers/sidecarproxy/builder/destinations_test.go

```diff
@@ -32,7 +32,7 @@ var (
 				},
 				Ports: map[string]*pbcatalog.WorkloadPort{
 					"tcp":  {Port: 7070, Protocol: pbcatalog.Protocol_PROTOCOL_TCP},
-					"tcp2": {Port: 8081, Protocol: pbcatalog.Protocol_PROTOCOL_TCP},
+					"grpc": {Port: 8081, Protocol: pbcatalog.Protocol_PROTOCOL_TCP},
```

Comment:
```suggestion
					"grpc": {Port: 8081, Protocol: pbcatalog.Protocol_PROTOCOL_GRPC},
```

Did you want to switch the protocol too?